samza-core/src/main/java/org/apache/samza/execution/JobGraphJsonGenerator.java

```diff
@@ -44,9 +45,9 @@
 /**
  * This class generates the JSON representation of the {@link JobGraph}.
  */
-/* package private */ class JobGraphJsonGenerator {
+public class JobGraphJsonGenerator {
```

Comment:
Since we are making them public, lets add some java docs for all the Pojo's and there use

Reply:
Thanks for the review! Added the java docs for those inner classes.